src/models/hooks/sessionReportPilot.test.js

```diff
@@ -478,6 +478,32 @@ describe('createGoalsForSessionRecipientsIfNecessary hook', () => {
     expect(mockSequelize.models.GoalCollaborator.update).toHaveBeenCalledTimes(0);
   });
 
+  it('doesn\'t fail when recipients is null', async () => {
```

Comment:
This test seems to pass with or without your change

Reply:
Hmm, so it does! 🤔

Reply:
Thanks. I think it looks okay now.